src/policy/fees.cpp

```diff
@@ -423,6 +424,10 @@ CFeeRate CBlockPolicyEstimator::estimateSmartFee(int confTarget, int *answerFoun
     if (confTarget <= 0 || (unsigned int)confTarget > feeStats.GetMaxConfirms())
         return CFeeRate(0);
 
+    // It's not possible to get reasonable estimates for confTarget of 1
+    if (confTarget == 1)
```

Comment:
@jtimon I wanted to leave the external behavior essentially unchanged.  For estimatefee, it often returned -1, so making it always do that would not be unexpected.  It would be very unexpected for estimatesmartfee to not return a reasonable answer.  Note that because of this the GUI changes are actually optional, but I think provide for a more clear experience.